### DISCUSSION TITLE
<select>

### DISCUSSION THREAD

**Discussion by Peter-Dobbins** [community/community#27337] [Other Feature Feedback, Questions, & Ideas]
If a background-colour is set for a select element, I cannot then change the min-height of that element. It just goes back to its original size. 

Anyone know why?

Cheers.
<img width="365" alt="Screenshot 2022-07-27 at 17 20 41" src="https://user-images.githubusercontent.com/109819737/181299603-c5ba52f5-335f-4371-9352-a3a39eb5e22a.png">

**Comment by sawyerbivens** [comment]
I can't seem to replicate this, are you using any kind of third party libraries or other CSS imports?

Try and check your inspect element and make sure your min-height tag isn't being overridden

![image](https://user-images.githubusercontent.com/48366954/181310098-0fd266dc-416f-46fa-8106-47299875d315.png)

**Reply by Peter-Dobbins** [reply]
Hey, thanks for replying.

I am using free code camp and was working on the first certification project. I tried it in an empty project still with the same results. I have attached images.

Any idea what is going wrong? 



<img width="1434" alt="Screenshot 2022-07-28 at 13 56 18" src="https://user-images.githubusercontent.com/109819737/181513316-59453ad0-f684-4a04-b782-bae63d75a461.png">
<img width="1372" alt="Screenshot 2022-07-28 at 13 56 38" src="https://user-images.githubusercontent.com/109819737/181513329-7ff1c546-9f41-4edd-82ab-38137ced38a9.png">
<img width="1430" alt="Screenshot 2022-07-28 at 13 57 37" src="https://user-images.githubusercontent.com/109819737/181513334-cd2b4bf6-8b94-4414-8bf5-7a7663e6b779.png">

**Reply by sawyerbivens** [reply]
Try and use the normal 'height' selector instead of the 'min-height' one maybe, I'll try to do some tests with your same HTML as soon as I get on a computer.

**Reply by Peter-Dobbins** [reply]
Strangely the height selector does nothing even when on its own, so I went back though my notes on free code camp and they used a min-height to do the job, but didn't colour it.

**Reply by sawyerbivens** [reply]
Just noticed that your `select` tag is a child of the `label tag`, try and switch to look something like this:
```html
<fieldset>
    <label class="block">Which option best describes your reason for contact?</label>
    <select class="block size" id="dropdown" required>
        <option value="">Select One</option>
        <option value="1">Music</option>
        <option value="2">Coding</option>
        <option value="3">Science and Tec</option>
    </select>
</fieldset>
```

**Reply by Peter-Dobbins** [reply]
Na, still giving me grief.

I've also tried all the units. I really am at a loss.
<img width="1408" alt="Screenshot 2022-07-28 at 16 02 10" src="https://user-images.githubusercontent.com/109819737/181571471-958c9ff9-0052-489f-bdb3-5e37317d7a8e.png">

**Reply by Peter-Dobbins** [reply]
Not unless its an issue with free code camp?

**Reply by Peter-Dobbins** [reply]
Nope. Strange. . . 

<img width="967" alt="Screenshot 2022-07-28 at 16 06 30" src="https://user-images.githubusercontent.com/109819737/181572277-c5555a37-2060-4b4f-8679-b4293530f702.png">

**Reply by Peter-Dobbins** [reply]
I use Safari. Could that be the issue?

**Reply by sawyerbivens** [reply]
Yea it must either be an issue with Freecodecamp or Safari because I've got it working with or without the background using a different online editor.
<img width="1428" alt="Screen Shot 2022-07-28 at 10 09 05 AM" src="https://user-images.githubusercontent.com/48366954/181572839-1ee473b5-a41d-4f0f-ad2a-1ab4574c472e.png">

**Reply by Peter-Dobbins** [reply]
Which browser and online editor do you use?

**Reply by sawyerbivens** [reply]
I'm on Google Chrome and am using [this](https://www.w3schools.com/html/tryit.asp) editor

**Reply by Peter-Dobbins** [reply]
Wicked. I will give it a go on the morrow. Thanks for your help man. Really appreciate it.

**Reply by Peter-Dobbins** [reply]
That link seems to be broken??

**Reply by sawyerbivens** [reply]
Of couse

**Reply by sawyerbivens** [reply]
> That link seems to be broken??

try this:
https://www.w3schools.com/html/tryit.asp?filename=tryhtml_default_default

**Reply by Peter-Dobbins** [reply]
Great! Thank you!

**Comment by FEsoftdev** [comment]
https://stackoverflow.com/questions/68264388/height-of-the-select-option-box
Please check this url.
This might be help 🙂

**Reply by Peter-Dobbins** [reply]
Hey, thanks for replying.

I have tried to use a class selector for one property and a type selector for another, then reversed it, and it seems background-color always trumps min-height. 

Any ideas why or what I can do to fix it?

<img width="1433" alt="Screenshot 2022-07-28 at 14 13 16" src="https://user-images.githubusercontent.com/109819737/181514391-b4d17141-573f-4f9f-91c9-0236c73fc53b.png">
<img width="1435" alt="Screenshot 2022-07-28 at 14 14 02" src="https://user-images.githubusercontent.com/109819737/181514407-66c3ea22-b823-46b1-b9ea-aaca01c06ef1.png">
<img width="1438" alt="Screenshot 2022-07-28 at 14 14 56" src="https://user-images.githubusercontent.com/109819737/181514412-5a8eb514-fcf9-4bed-be6a-27432feeda32.png">
